pkg/parca/parca.go

```diff
@@ -75,6 +81,12 @@ type Flags struct {
 
 	UpstreamDebuginfodServer     string        `default:"https://debuginfod.systemtap.org" help:"Upstream private/public server for debuginfod files. Defaults to https://debuginfod.systemtap.org."`
 	DebugInfodHTTPRequestTimeout time.Duration `default:"5m" help:"Timeout duration for HTTP request to upstream debuginfod server. Defaults to 5m"`
+
+	StoreAddress       string `kong:"help='gRPC address to send profiles and symbols to.'"`
+	BearerToken        string `kong:"help='Bearer token to authenticate with store.'"`
+	BearerTokenFile    string `kong:"help='File to read bearer token from to authenticate with store.'"`
+	Insecure           bool   `kong:"help='Send gRPC requests via plaintext instead of TLS.'"`
+	InsecureSkipVerify bool   `kong:"help='Skip TLS certificate verification.'"`
```

Comment:
Be able to specify external labels would be better. That way I can filter series by external labels.

Reply:
💯

Reply:
Yes, totally! 
That will be a follow-up PR.